widget/popup.go

```diff
@@ -69,6 +69,15 @@ func (p *PopUp) ShowAtPosition(pos fyne.Position) {
 	p.Show()
 }
 
+// ShowAtRelativePosition shows this pop-up at the given position relative to stated object.
+//
+// Since 2.4
+func (p *PopUp) ShowAtRelativePosition(rel fyne.Position, to fyne.CanvasObject) {
+	withRelativePosition(rel, to, func(pos fyne.Position) {
+		p.ShowAtPosition(pos)
+	})
```

Comment:
You should be able to drop the wrapper function here.
```suggestion
	withRelativePosition(rel, to, p.ShowAtPosition)
```